Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -213,27 +213,217 @@
   - SurveillanceWirelessCameraMonitorCircuitboard
   - ComputerTelevisionCircuitboard
 
-#- type: technology
-#   name: "ballistic technology"
-#   id: BallisticTechnology
-#   description: Just a fancy term for guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Pistols/mk58.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
-#
-# - type: technology
-#   name: "direct energy technology"
-#   id: DirectEnergyTechnology
-#   description: Basically laser guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Battery/taser.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
+- type: technology
+  name: technologies-security-basic
+  id: BasicSecurityTechnology
+  description: technologies-security-basic-description
+  icon:
+    sprite: Objects/Misc/handcuffs.rsi
+    state: handcuffs
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicResearch
+  unlockedRecipes:
+    - Handcuffs
+    - Zipties
+    - RiotShield
+    - RiotLaserShield
+    - RiotBulletShield
+    - ClothingGlassesEyesSecurity
+    - FlashlightSeclite
+
+- type: technology
+  name: technologies-forensics-equipment
+  id: ForensicsEquipmentTechnology
+  description: technologies-forensics-equipment-description
+  icon:
+    sprite: Objects/Devices/forensic_scanner.rsi
+    state: forensicnew
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - ForensicScanner
+    - ForensicPad
+
+- type: technology
+  name: technologies-speed-loader
+  id: SpeedLoaderTechnology
+  description: technologies-speed-loader-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_speed_loader.rsi
+    state: base-6
+  requiredPoints: 5000
+  requiredTechnologies:
+    - ForensicsEquipmentTechnology
+    - BallisticMagazinesTechnology
+  unlockedRecipes:
+    - SpeedLoaderRubber
+    - SpeedLoaderMagnum
+    - SpeedLoaderHighVelocity
```

Comment:
i don't get why speedloaders are their own tech instead of being with the magazines (rubber with nonlethal, magnum with reg. bullets, high velocity with other high velocity)

Reply:
you asked me to do it last time